tests/test_delb_plugin.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+
+from delb import Document
+from snakesist import ExistClient
+
+
+@pytest.mark.usefixtures("db")
+def test_delete_document():
+    url = "existdb://admin:@localhost:8080/test_collection/test_document.xml"
+    document = Document(url)
+    document.existdb_delete()
+
+
+@pytest.mark.usefixtures("db")
+def test_load_document_from_url():
+    url = "existdb://admin:@localhost:8080/test_collection/test_document.xml"
+    document = Document(url)
+
+    assert document.config.source_url == url
+    assert isinstance(document.config.existdb.abs_id, str)
+    assert isinstance(document.config.existdb.client, ExistClient)
+    assert document.existdb_collection == "/test_collection/"
+    assert document.existdb_filename == "test_document.xml"
+
+
+@pytest.mark.usefixtures("db")
+def test_load_document_with_client():
+    exist_client = ExistClient(host="localhost", port=8080, user="admin", password="")
+    document = Document(
+        "/test_collection/test_document.xml", existdb_client=exist_client
```

Comment:
this might result in ambiguities. let's say the document would be initialized with an additional client for another storage backend that could also make sense of `"/test_collection/test_document.xml"`.

Reply:
I'm probably not picturing this correctly, but I'm not sure I understand where this would be ambiguous. What would be the expected behavior from a document with two storage backends and where would it go wrong in this case?